systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -1033,6 +1134,20 @@ void testDynamicallySetUnknownKafkaLogger() {
     }
 
     @ParallelNamespaceTest
+    @TestDoc(
+        description = @Desc("Test the dynamic setting of an unknown Kafka logger value without triggering a rolling update."),
+        steps = {
+            @Step(value = "Create test storage and resources with required namespaces and node pools.", expected = "Resources are created with the correct configuration."),
```

Comment:
As before - the information about test storage is not useful for the overall test case. And what is correct configuration? Is there something changed?